doc/releases/quincy.rst

```diff
@@ -5,6 +5,537 @@ Quincy
+* mgr/dashboard: refactor dashboard cephadm e2e tests (`pr#48432 <https://github.com/ceph/ceph/pull/48432>`_, Nizamudeen A)
+* mgr/dashboard: Replace vonage-status-panel with native grafana stat panel (`pr#50043 <https://github.com/ceph/ceph/pull/50043>`_, Aashish Sharma)
+* mgr/dashboard: rgw server side encryption config values set to wrong daemon (`pr#49724 <https://github.com/ceph/ceph/pull/49724>`_, Aashish Sharma)
+* mgr/dashboard: Unable to change rgw subuser permission (`pr#48440 <https://github.com/ceph/ceph/pull/48440>`_, Aashish Sharma)
+* mgr/dashboard: upgrade to angular 13, bootstrap 5 and jest 28 (`pr#50124 <https://github.com/ceph/ceph/pull/50124>`_, Nizamudeen A, Bryan Montalvan)
+* mgr/nfs: add sectype option (`pr#48531 <https://github.com/ceph/ceph/pull/48531>`_, John Mulligan)
+* mgr/nfs: handle bad cluster name during info command (`pr#49654 <https://github.com/ceph/ceph/pull/49654>`_, Dhairya Parmar)
+* mgr/orchestrator: fix upgrade status help message (`pr#49855 <https://github.com/ceph/ceph/pull/49855>`_, Adam King)
+* mgr/prometheus: change pg_repaired_objects name to pool_repaired_objects (`pr#48438 <https://github.com/ceph/ceph/pull/48438>`_, Pere Diaz Bou)
+* mgr/prometheus: export zero valued pg state metrics (`pr#49787 <https://github.com/ceph/ceph/pull/49787>`_, Avan Thakkar)
+* mgr/prometheus: expose daemon health metrics (`pr#49519 <https://github.com/ceph/ceph/pull/49519>`_, Pere Diaz Bou)
+* mgr/prometheus: expose repaired pgs metrics (`pr#48204 <https://github.com/ceph/ceph/pull/48204>`_, Pere Diaz Bou)
+* mgr/prometheus: fix module crash when trying to collect OSDs metrics (`pr#49930 <https://github.com/ceph/ceph/pull/49930>`_, Redouane Kachach)
+* mgr/prometheus: use vendored "packaging" instead (`pr#49698 <https://github.com/ceph/ceph/pull/49698>`_, Kefu Chai, Matan Breizman)
+* mgr/rbd_support: avoid wedging the task queue if pool is removed (`pr#49057 <https://github.com/ceph/ceph/pull/49057>`_, Ilya Dryomov)
+* mgr/rbd_support: remove localized schedule option during module startup (`pr#49649 <https://github.com/ceph/ceph/pull/49649>`_, Ramana Raja)
+* mgr/rook: Device inventory (`pr#49877 <https://github.com/ceph/ceph/pull/49877>`_, Juan Miguel Olmo Martínez)
+* mgr/rook: fix error when trying to get the list of nfs services (`pr#48199 <https://github.com/ceph/ceph/pull/48199>`_, Juan Miguel Olmo Martínez)
+* mgr/rook:NFSRados constructor expects type of rados as a parameter instead of MgrModule (`pr#48830 <https://github.com/ceph/ceph/pull/48830>`_, Ben Gao)
+* mgr/snap_schedule: remove subvol interface (`pr#48222 <https://github.com/ceph/ceph/pull/48222>`_, Milind Changire)
+* mgr/telemetry: add `basic_pool_options_bluestore` collection (`pr#49414 <https://github.com/ceph/ceph/pull/49414>`_, Laura Flores)
+* mgr/telemetry: handle daemons with complex ids (`pr#48283 <https://github.com/ceph/ceph/pull/48283>`_, Laura Flores)
```

Comment:
Remove this. It was introduced in 17.2.5 as a cherry-pick, so the script probably picked up the merge commit when it shouldn't have.

Reply:
Line 438